aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
 
 ## Blazor
 
-We hope to have the Blazor section of this article updated soon.<!--In the meantime, see the following resources: -->
+<!-- UPDATE 8.0 Add API cross-links after RTM -->
 
-### Adopt .NET 8 features
+The following migration scenarios are covered:
 
-After following the guidance earlier in this article to update an app to 8.0, adopt specific features by following the links in <xref:aspnetcore-8#blazor>.
+* [Update a Blazor Server app](#update-a-blazor-server-app)
+* [Add interactive WebAssembly components to a Blazor Server app](#add-interactive-webassembly-components-to-a-blazor-server-app)
```

Comment:
This isn't really a migration scenario, so I think we should move this content elsewhere. I think what we should do is generalize the [Integrate components](https://learn.microsoft.com/aspnet/core/blazor/components/integration) doc a bit and move this content there. Right now the Integrate component doc focuses specifically on adding components to an MVC or Razor Pages app, but we also need to document how to add Blazor support to _any_ ASP.NET Core app, even an [API project](https://github.com/dotnet/aspnetcore/issues/51384#issuecomment-1766247037) that currently has no UI. This content should include how to setup a client project. The easiest way to do this might be to use the Blazor WebAssembly Standalone project template and then modify it as needed.